Resources/Prototypes/_Funkystation/Entities/Objects/Misc/crystals.yml

```
@@ -0,0 +1,105 @@
+  components:
+  - type: Sprite
+    sprite: _Funkystation/Objects/Power/crystal_cell.rsi
+    layers:
+    - state: crystal_cell
+  - type: Battery
+    maxCharge: 25000
+    startingCharge: 25000
+    pricePerJoule: 0.01
+  - type: Tag
+    tags:
+    - NoRecharge
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)